mta/data/analysis.json

```diff
@@ -4,15 +4,34 @@
     "source": "source_code",
     "targets": [
       "cloud-readiness"
-    ],
-    "story_points": 5
+    ]
   },
   {
     "app_name": "tackle-testapp",
     "source": "source_code",
     "targets": [
       "cloud-readiness"
-    ],
-    "story_points": 1
+    ]
+  },
+  {
+    "app_name": "tackle-testapp",
+    "source": "binary",
+    "mode": {
+      "binary": true
```

Comment:
mode and source are same . why do we need both ?

analysis_data.get("source") will give data like "source" or "binary"

Reply:
`mode` is needed for task creation. It contains a different combination of binary, withDeps, artifact, etc. values depending on the analysis type